src/Apps/Order/Routes/Status/index.tsx

```diff
@@ -85,7 +85,7 @@ export class StatusRoute extends Component<StatusProps> {
                 {buyerRejectedOffer ? (
                   <Button
                     onClick={() => {
-                      this.props.router.push(`/`)
+                      window.location.href = "/"
```

Comment:
We're inside the order app and trying to transition to the main page, which is a different app with a different entry point so it can't be done within found as far as I can tell. Maybe we can set up the order app to trigger a normal page change if the new route doesn't start with `/order`?

Reply:
Ahh I see, that makes perfect sense and in which case this is the correct approach. If it was routing code all the way down (one day..) this would break things, but here we're jumping out. 👍